qt_ui/windows/mission/flight/QFlightCreator.py

```diff
@@ -34,6 +35,13 @@ def __init__(self, game: Game, package: Package) -> None:
 
         layout = QVBoxLayout()
 
+        self.flight_name = QLineEdit()
+        # todo: auto generate a friendly flight name
+        # option 1: auto-increment name
+        # option 2: auto-increment by global flight number
+        layout.addLayout(QLabeledWidget("Name:", self.flight_name))
+        # todo: also on the edit screen
```

Comment:
Edit screen is done, so ignore

But auto-generating cool sounding flight names - any sensible strategies?

Reply:
Use the DCS callsign name? We can't *set* those, by pydcs does configure them and we can read them. Going outside what DCS provides probably isn't great since then you need to remember two callsigns (DCS only knows how to pronounce the ones it knows, so AWACS/ATC/whatever comms wills always call you Enfield or whatever).

Reply:
The goal of the feature is to allow multiplayer people find their assignment slots quickly and rapidly in large groups. It will won't be a replacement for the built-in DCS commands.

Using your thought process, perhaps labelling this as `Group Label` would be more true to the _intent_?

Reply:
I don't follow. Is Enfield 1 not findable in a large group?